Proiect_PAO/src/aplicatieBancara/Card/Card.java

```diff
@@ -0,0 +1,108 @@
+package aplicatieBancara.Card;
+
+import aplicatieBancara.Cont.Cont;
+import aplicatieBancara.TipCard;
+
+import java.nio.charset.StandardCharsets;
+import java.util.*;
+
+
+public abstract class Card {
+
+    protected final int cardId;
+    protected final int CVV;
+    protected String numar;
+    protected String IBAN;
+    protected double sold;
+    protected Cont cont;
+    protected final Date dataExpirare;
+    //protected String PIN;
+
+
+    static private final HashSet<String> usedNumbers = new HashSet<String>();
```

Comment:
poti lasa doar new HashSet<>()

Reply:
small nitpick, ar fi dragut sa nu combinam si engleza si romana in aplicatie, am vazut ca ai ales in romana in general, poti schimba si aici numele setului in ceva in romana, un prim exemplu la care ma gandesc: numereCardAlocate